cmd/kubeadm/app/images/images.go

```diff
@@ -121,3 +122,12 @@ func GetControlPlaneImages(cfg *kubeadmapi.ClusterConfiguration) []string {
 
 	return imgs
 }
+
+// GetPauseImage returns the image for the "pause" container
+func GetPauseImage(cfg *kubeadmapi.ClusterConfiguration) string {
+	//If user has configured the cluster to use a different image repository, use that for the pause image.
+	if cfg.ImageRepository != kubeadmapiv1beta2.DefaultImageRepository {
+		return GetGenericImage(cfg.ImageRepository, "pause", constants.PauseVersion)
+	}
+	return GetGenericImage("k8s.gcr.io", "pause", constants.PauseVersion)
+}
```

Comment:
the one from _unix.go

```
// GetPauseImage returns the image for the "pause" container
func GetPauseImage(cfg *kubeadmapi.ClusterConfiguration) string {
	return GetGenericImage(cfg.ImageRepository, "pause", constants.PauseVersion)
}
```

Reply:
Looks good to me.
It seems to be correct as well.